### PR TITLE
Control the number of drift chambers

### PR DESCRIPTION
**Pull request by myliu-hub**
Hi, @mirguest , @ihepzhangyao ,I have improved the number control of the drift chamber，among them, the meaning of the parameter is：
 0:inner chamber + outer chamber;
 1:only inner chamber;
 2:only outer chamber; 
I hope you will provide valuable comments on the changes I made, thank you.

### CODE REVIEW

**Comment by mirguest** [comment]
Hi @myliu-hub ,
I think the `DC_num_chamber` is quite confusing. It makes me think the number of chambers enabled. So 0 means no chambers, 1 means one chamber enabled and 2 means two chambers enabled.

So I think it would be necessary to find a better name to indicate such flag.

In my option, you can define more explicit variables such as `DC_inner_chamber_enabled` and `DC_outer_chamber_enabled`. Use 1 to enable it and use 0 to disable it. Even though you have an extra variable defined, but it is more clear.

Tao

**Comment by myliu-hub** [comment]
Hi, @mirguest and @ihepzhangyao ,I have made changes based on the teacher's suggestion, please check it, thank you.

File: Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml [hunk 1]
```diff
@@ -44,6 +44,10 @@
     <constant name="DC_Endcap_rmax" value="SDT_radius_max"/>
     <constant name="DC_Endcap_dz" value="DC_Endcap_z"/>
 
+    <constant name="DC_inner_chamber_enabled" value="0"/>
+    <constant name="DC_outer_chamber_enabled" value="0"/>
+    <constant name="DC_all_chamber_enabled" value="1"/>
```
Comment (mirguest) [inline Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml:49]: I think the first two variables are enough, the third one is not necessary.

**Review by mirguest — CHANGES_REQUESTED** [review]

File: Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml (hunk 1 above)
Comment (myliu-hub) [reply Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml]: I have changed the control parameter to oneChange the control parameter to one, please check it, thank you.

**Comment by mirguest** [comment]
Hi @myliu-hub , why close this PR?
Tao

**Comment by myliu-hub** [comment]
> Hi @myliu-hub , why close this PR?
> Tao

Sorry, it was my operation error.I have reopened.

**Comment by mirguest** [comment]
Hi @myliu-hub ,
Do you finish this PR? If you have done, I will start the review.
Tao

**Comment by myliu-hub** [comment]
> Hi @myliu-hub ,
> Do you finish this PR? If you have done, I will start the review.
> Tao

Yes, I have done it.

**Review by mirguest — APPROVED** [review]
Looks good. Thanks.
Tao

**Pull request merged**